data/data/agent/systemd/units/agent-interactive-console.service

```diff
@@ -1,7 +1,7 @@
 [Unit]
 Description=Get interactive user configuration at boot
 After=network-pre.target NetworkManager.service pre-network-manager-config.service selinux.service
-Before=network.target network.service agent.service NetworkManager-wait-online.service
+Before=network.target network.service agent.service node-zero.service NetworkManager-wait-online.service
```

Comment:
I'm surprised `getty@tty1.service` is not on this list

Reply:
Actually, we should use `getty-pre.target` here.
https://www.freedesktop.org/software/systemd/man/systemd.special.html#getty-pre.target

Reply:
Is `network.service` a real thing?

I thought for a minute that we should use `Before=getty-pre.target` (https://www.freedesktop.org/software/systemd/man/systemd.special.html#getty-pre.target), but that will block _all_ consoles and we are only displaying on one console, so I think `Before=getty@tty1.service` is the right thing.